src/main/java/com/thealgorithms/dynamicprogramming/SumOfSubset.java

```diff
@@ -1,18 +1,37 @@
 package com.thealgorithms.dynamicprogramming;
 
-public class SumOfSubset {
+public final class SumOfSubset {
```

Comment:
Please document this class (cf. [previous review](https://github.com/TheAlgorithms/Java/pull/5030#pullrequestreview-1846442010)).